### PR TITLE
Update documentation for linear regression

### PR DESCRIPTION
**Pull request by farzadras**
I updated the following file regarding MADLIB-664:
src/ports/postgres/modules/regress/linear.sql_in

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
```
Comment (haradh1) [inline Makefile]: Don't include this, as this is a generated file.

Comment (farzadras) [inline Makefile]: Right! I did't figure out how to exclude Makefile when doing the pull request :)

**Comment by fschopp** [comment]
Could you remove the Makefile and rebase into one commit?

**Comment by farzadras** [comment]
Sure. I'll keep you posted.

**Comment by farzadras** [comment]
Done!

**Comment by farzadras** [comment]
Accidentally pressed the close button. Still new at Github :p

**Comment by cwelton** [comment]
Looks good!

**Comment by fschopp** [comment]
I'll do the merge later in the day.

**Comment by fschopp** [comment]
Merged with commit f88432b.

**Pull request closed without merging**